docker/deployed/docker-compose.observability.yml

```diff
@@ -20,4 +20,4 @@ services:
       - GF_USERS_ALLOW_SIGN_UP=false
     restart: unless-stopped
     ports:
-      - "127.0.0.1:3000:3000"
+      - "0.0.0.0:3000:3000"
```

Comment:
Bind to `0.0.0.0` so it's actually listening for external requests (if firewall config allows).